templates/web/pages/user_form/user_form_page.tt.html

```diff
@@ -97,6 +99,17 @@
                 [% END %]
 
             [% END %]
+        </form>
+    </div>
+    <!-- show video about account creation in Pro Platform -->
```

Comment:
We don't want the video displayed when we are not on the pro platform, is that the case?